django/core/validators.py

```diff
@@ -136,6 +136,12 @@ def __call__(self, value):
                     raise ValidationError(self.message, code=self.code)
             url = value
 
+        if len(urlsplit(value).netloc) > 253:
```

Comment:
Why 253? My intuition would be 255 minus 1 for dot & 1 for tld but please confirm with a comment as above. Please wrap comments at 79 chars. Not sure it's important to mention the bit about why it can't be done in the regular expression.